draft-ietf-mmusic-sdp-uks.md

```diff
@@ -336,14 +342,56 @@ Section 3 of {{!TLS13}}:
    } ExternalIdentityHash;
 ~~~
 
-A WebRTC identity assertion is provided as a JSON {{?JSON=RFC8259}} object that
-is encoded into a JSON text.  The resulting string is then encoded using UTF-8
-{{!UTF8=RFC3629}}.  The content of the `external_id_hash` extension are produced
-by hashing the resulting octets with SHA-256 {{!SHA=RFC6234}}.  This produces
-the 32 octets of the `binding_hash` parameter, which is the sole contents of the
+Where an identity assertion has been asserted by a peer, this extension includes
+a SHA-256 hash of the assertion.  An empty value is used to indicates support for
+the extension.
+
+Note:
+
+: For both types of identity assertion, if SHA-256 should prove to be inadequate
+  at some point in the future (see {{?AGILITY=RFC7696}}), a new TLS extension
+  can be defined that uses a different hash function.
+
+Identity bindings might be provided by only one peer.  An endpoint that does not
+produce an identity binding MUST generate an empty `external_id_hash` extension
+in its ClientHello.  An empty extension has a zero-length binding_hash field.
+This allows its peer to include a hash of its identity binding.  An endpoint
+without an identity binding MUST include an empty `external_id_hash` extension
+in its ServerHello or EncryptedExtensions message, to indicate support for the
```

Comment:
as does this for the SH/EE.
I don't have any better suggestions than "endpoint acting as TLS {client|server}", though.